CMakeLists.txt

```diff
@@ -190,6 +208,18 @@ mark_as_advanced( KWIVER_INSTALL_SET_UP_SCRIPT )
 if( KWIVER_INSTALL_SET_UP_SCRIPT )
   install( PROGRAMS   ${KWIVER_SETUP_SCRIPT_FILE}
     DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  if(WIN32)
+    install( PROGRAMS   ${KWIVER_SETUP_BATCH_FILE}
+      DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  endif()
+endif()
+
+if ( fletch_DIR )
```

Comment:
You might want to use `fletch_FOUND` here instead.  The user can set `fletch_DIR` and it might be set but invalid.  `fletch_FOUND` indicates that `find_package` found what it was looking for.